src/widgetastic_patternfly/__init__.py

```diff
@@ -502,6 +503,15 @@ def currently_selected(self):
         """A property to return the currently selected menu item"""
         return [self.browser.text(el) for el in self.browser.elements(self.CURRENTLY_SELECTED)]
 
+    @property
+    def all_options(self):
+        """A property to return the list of options available in the BootstrapNav"""
+        b = self.browser
```

Comment:
Please avoid catching in a variable if you are not using it multiple time.

Reply:
this is no longer valid as i am using `b` multiple times now.

Reply:
please manage return in a single line

Reply:
Done.